truffleruby/src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -51,8 +51,7 @@
     public static final String RUBY_VERSION = "2.3.1";
     public static final int    RUBY_REVISION = 0;
     public static final String COMPILE_DATE = "2017";
-    // TODO (pitr-ch 11-Jan-2017): use our value when we are recognized by rubygems
```

Comment:
Specific workarounds are better than waiting for a new RubyGems version (but we should fix upstream as well obviously).
We can also patch our own Rubygems of course.